Loop/View Controllers/CarbEntryViewController.swift

```diff
@@ -179,6 +198,23 @@ final class CarbEntryViewController: LoopChartsTableViewController, Identifiable
     }
 
     private var foodKeyboard: EmojiInputController!
+    
+    private func updateDisplayAccurateCarbEntryWarning() {
+        deviceManager.loopManager.getLoopState { [weak self] (_, state) in
+            self?.shouldDisplayAccurateCarbEntryWarning = state.didInsulinCounteractionEffectsReachThreshold(HKQuantity(unit: GlucoseEffectVelocity.unit, doubleValue: 3), startDate: Date().addingTimeInterval(.minutes(-20)), endDate: Date())
```

Comment:
[nit] Consider pulling out the `3` and the `20` and putting them into well-named constants for visibility.